backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -128,4 +131,32 @@ class EakApplicationMutationService {
             }
         }
     }
+
+    @GraphQLDescription("Updates a note of an application")
+    fun updateApplicationNote(
+        applicationId: Int,
+        noteText: String,
+        dfe: DataFetchingEnvironment
+    ): String {
+        val context = dfe.getContext<GraphQLContext>()
+        val jwtPayload = context.enforceSignedIn()
+
+        return transaction {
+            val application = ApplicationEntity.findById(applicationId) ?: throw UnauthorizedException()
+            // We throw an UnauthorizedException here, as we do not know whether there was an application with id
+            // `applicationId` and whether this application was contained in the user's project & region.
+            if (noteText.length > NOTE_MAX_CHARS) {
+                throw InvalidNoteSizeException(NOTE_MAX_CHARS)
+            }
+            val user = AdministratorEntity.findById(jwtPayload.adminId)
+                ?: throw UnauthorizedException()
+
+            if (!mayUpdateApplicationsInRegion(user, application.regionId.value)) {
+                throw ForbiddenException()
+            }
+
+            ApplicationRepository.updateApplicationNote(applicationId, noteText)
+            return@transaction noteText
```

Comment:
It seems quite hacky to me that we return the note again.

Reply:
alright i reverted it to true (success)